### DISCUSSION TITLE
2007/06/28/%E5%87%A0%E7%A7%8D%E7%AE%80%E5%8D%95%E7%9A%84%E6%B5%8B%E8%AF%95%E7%A8%8B%E5%BA%8F%E6%B5%81%E7%A8%8B%E6%A8%A1%E5%9E%8B

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#168] [Announcements]
# 2007/06/28/%E5%87%A0%E7%A7%8D%E7%AE%80%E5%8D%95%E7%9A%84%E6%B5%8B%E8%AF%95%E7%A8%8B%E5%BA%8F%E6%B5%81%E7%A8%8B%E6%A8%A1%E5%9E%8B

大多数测试程序主要步骤就是以下几步：采集数据、处理数据、显示数据、保存数据。这几个步骤可以顺序执行。在一次实验中，通常要多次循环这一过程，因此，这种测试程序的模型如图1所示。图1中最后一个子 VI 是用来判断实验是否结束，是否进行下一次循环用的。在这个模型中，各个程序模块是单线程顺序执行的，它的好处是程序逻辑简单，容易设计和理解。

https://blog.qizhen.xyz/2007/06/28/%E5%87%A0%E7%A7%8D%E7%AE%80%E5%8D%95%E7%9A%84%E6%B5%8B%E8%AF%95%E7%A8%8B%E5%BA%8F%E6%B5%81%E7%A8%8B%E6%A8%A1%E5%9E%8B

**Comment by ruanqizhen** [comment]
之前博客下的留言：

geon
2007年12月17日 21:41 编辑
头一个（或几个）数据无效的问题确实是使用移位寄存器的最大缺点
